guides/asynchronous-tasks/readme.md

```diff
@@ -183,9 +183,9 @@ barrier.wait
 
 ## Stopping a Task
 
-When a task completes execution, it will enter
+When a task completes execution, it will enter the `complete` state (or the `failed` state if it raises an unhandled exception).
 
-There are various situations where you may want to stop a task ({ruby Async::Task#stop}). The most common case is shutting down a server, but other important situations exist, e.g. you may fan out multiple (10s, 100s) of requests, wait for a subset to complete (e.g. the first 5 or all those that complete within a given deadline), and then stop (terminate/cancel) the remaining operations.
+There are various situations where you may want to stop a task ({ruby Async::Task#stop}) before it completes. The most common case is shutting down a server, but other important situations exist, e.g. you may fan out multiple (10s, 100s) of requests, wait for a subset to complete (e.g. the first 5 or all those that complete within a given deadline), and then stop (terminate/cancel) the remaining operations.
 
 Using the above program as an example, we can 
```

Comment:
Also stopped half way through a sentence lol.

Reply:
I took a shot at finishing the sentence in the follow-up. :)